README.md

```diff
@@ -34,7 +34,7 @@ helm install stable/nginx-ingress --namespace example
 3. Add the helm repository so that chart dependencies can be pulled:
 
 ```bash
-helm repo add alfresco-test https://alfresco.github.io/charts-test/incubator
+helm repo add alfresco-test https://alfresco.github.io/charts/incubator
```

Comment:
Actually, looking closer, I think we wanted to consistently name this repo alfresco-incubator everywhere, right?

Reply:
yes, calling it alfresco-test seems inconsistent

Reply:
Since Sergiu should be home I've updated the name.

Reply:
Dudes - the URL is http://kubernetes-charts.alfresco.com/incubator not the github one. Sorry :-(